Source/DafnyCore/Triggers/QuantifierSplitter.cs

```diff
@@ -112,8 +112,7 @@ protected override void VisitOneExpr(Expression expr) {
         Contract.Assert(quantifier.SplitQuantifier == null);
         if (!splits.ContainsKey(quantifier) && AllowsSplitting(quantifier)) {
           splits[quantifier] = SplitQuantifier(quantifier).Select(e => {
-            e.SetTok(new AutoGeneratedOrigin(e.Tok));
-            e.Origin = e.Origin.MakeAutoGenerated();
+            e.SetOrigin(new AutoGeneratedOrigin(e.Origin));
```

Comment:
Interesting that MakeAutoGenerated is no longer necessary.

Reply:
It was being done twice before. `MakeAutoGenerated` does the same as `new AutoGeneratedOrigin(`